share/spice/public_holidays/record.handlebars

```diff
@@ -3,8 +3,8 @@
         {{#each holidays}}
             <tr class="record__row  {{#if ../meta.options.rowHighlight}}record__row--highlight{{/if}}">
                 <td class="record__cell  record__cell--key">{{date}}</td>                
-                <td class="record__cell  record__cell--key">{{name}}</td>                
-                <td class="record__cell  record__cell--key">{{notes}}</td>                
+                <td class="record__cell  record__cell--value">{{name}}</td>
+                <td class="record__cell  record__cell--value">{{notes}}</td>
```

Comment:
hmm that's odd -- the table should be filling the width there and letting the 2nd column stretch out.

What browser is that?